### PR TITLE
Adopt PurchasesHybridCommon 3.2.0

### PR DESCRIPTION
**Pull request by taquitos**

### CODE REVIEW

File: src/ios/PurchasesPlugin.swift [hunk 1]
```diff
@@ -186,48 +191,42 @@ public class CDVPurchasesPlugin : CDVPlugin {
...
     }
 
     @objc(isAnonymous:)
     func isAnonymous(command: CDVInvokedUrlCommand) {
-        let result = CDVPluginResult(status: .ok, messageAs: RCCommonFunctionality.isAnonymous())
+        let result = CDVPluginResult(status: .ok, messageAs: CommonFunctionality.isAnonymous)
         self.commandDelegate.send(result, callbackId: command.callbackId)
     }
 
-    // NOTE: This method crashes now that it's in Swift. The underlying method call relies on an implicit cast
```
Comment (taquitos) [inline src/ios/PurchasesPlugin.swift:199]: This is actually still a problem, just slightly different problem now: https://github.com/RevenueCat/purchases-hybrid-common/pull/159

File: src/ios/PurchasesPlugin.swift [hunk 2]
```diff
@@ -186,48 +191,42 @@ public class CDVPurchasesPlugin : CDVPlugin {
...
     }
 
     @objc(presentCodeRedemptionSheet:)
     func presentCodeRedemptionSheet(command: CDVInvokedUrlCommand) {
         if #available(iOS 14.0, *) {
-            RCCommonFunctionality.presentCodeRedemptionSheet()
+            CommonFunctionality.presentCodeRedemptionSheet()
         } else {
-            Logger.warn("[Purchases] Warning: tried to present codeRedemptionSheet, but it's only available on iOS 14.0 or greater.")
+            NSLog("%@", "[Purchases] Warning: tried to present codeRedemptionSheet, but it's only available on iOS 14.0 or greater.")
```
Comment (joshdholtz) [inline src/ios/PurchasesPlugin.swift:229]: Is this supposed to be `NSLog`? 🤔

**Review by joshdholtz — COMMENTED** [review]
Niiiiccce! I left one comment about a wild `NSLog` that appeared 🤷‍♂️ 

Is there also an Android component to adopting hybrid common 3.2.0?

File: plugin.xml [hunk 3]
```diff
@@ -32,7 +33,7 @@
                 <source url="https://github.com/CocoaPods/Specs.git"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="PurchasesHybridCommon" spec="1.10.0"/>
+                <pod name="PurchasesHybridCommon" spec="3.2.0"/>
```
Comment (aboedo) [inline plugin.xml:36]: note: cordova for Android will be broken until the android side of this is done too

**Review by aboedo — APPROVED** [review]

File: src/ios/PurchasesPlugin.swift (hunk 2 above)
Comment (taquitos) [reply src/ios/PurchasesPlugin.swift:229]: > Niiiiccce! I left one comment about a wild `NSLog` that appeared 🤷‍♂️
> 
> Is there also an Android component to adopting hybrid common 3.2.0?

There are a couple jiras for it, but I haven't worked through them yet, focusing on iOS for now while I learn.

Comment (taquitos) [reply src/ios/PurchasesPlugin.swift:229]: Re: NSLog- that's how we do logging currently 😄 https://github.com/RevenueCat/purchases-ios/blob/f1044bce14655a7b89423dd33e508963275ad16d/Sources/Logging/Logger.swift#L68

**Pull request merged**